### ISSUE TITLE
Enclosing table names in square brackets breaks Dapper.Contrib

### ISSUE THREAD

**Issue by Stiles-X** [StackExchange/Dapper#1584]
`[Dapper.Contrib.Extensions.Table("[Admin.Office]")]` and alternately `[System.ComponentModel.DataAnnotations.Schema.Table("[Admin.Office]")]` both does not work.

It results in `SqlException: Invalid object name 'Admin.Office'`.

The square brackets seems to have disappeared. #336 is related but only for column names, not table names.  
A failed attempt escaping the brackets using "\\" results in `SqlException: Incorrect syntax near '\'`.

Details: 
* Dapper and Dapper.Contrib 2.0.78. 
* .NET 5 C# ClassLibrary with NullableEnabled.
* SqlServer 2019 Express Edition with `System.Data.SqlClient` connection nuget package.

**Comment by Stiles-X** [comment]
Upon further research, more things are broken. It seems I have got the wrong conclusion. Invalid object name. Not table. So this is a different problem. I'm closing this until I have a better grip on what's happening.

**Comment by Stiles-X** [comment]
With help from [this msdn forum on this exception](https://social.msdn.microsoft.com/Forums/en-US/b23617e2-a4a1-44c7-9e19-f03bba7eb144/systemdatasqlclientsqlexception-invalid-object-name) and [check if sql connection is successful or not](https://social.msdn.microsoft.com/Forums/vstudio/en-US/7179cdcc-fa85-46bb-bd92-374361c569e5/how-i-check-sql-connection-is-successful-or-not).

Confirms that the connection wasn't working at all. I hope this helps future wanderers solve their issue.

TL;DR: Connection wasn't working. Not Dapper issue.
